baha/6D_Bellman-Ford.py

```diff
@@ -0,0 +1,52 @@
+from collections import defaultdict
+
+
+class Graph:
+    def __init__(self, nodes_no, edges_no):
+        self._graph = defaultdict(list)
+        self.nodes_no = nodes_no
+        self.edges_no = edges_no
```

Comment:
```suggestion
        self._nodes_no = nodes_no
        self._edges_no = edges_no
```

It seems like all fields can be private.